src/components/Article.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import Viewcomments from "./Viewcomments";
+
+function Article() {
+const { article_id } = useParams();
+const [article, setArticle] = useState({}); 
+const [isLoading, setIsLoading] = useState(true);
+useEffect(() => {
+    setIsLoading(true);
+    fetch(`https://news-be-heroku.herokuapp.com/api/articles/${article_id}`)
```

Comment:
Please be wary of using fetch for this, we moved on to axios as its so much better for error handling (which you've not done yet)